packages/@ember/-internals/glimmer/lib/helpers/modifier.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ @module ember
+ */
+
+/**
+ Use the `{{modifier}}` helper to create contextual modifier so
+ that it can be passed around as first-class values in templates.
+
+ ```handlebars
+ {{#let (modifier "click-outside" click=this.submit) as |on-click-outside|}}
+
+   <MyComponent {{on-click-outside}} />
+
+   {{!-- these will pass the modifier itself into the component, instead of invoking it now --}}
+
+   <MyComponent @modifier={{modifier on-click-outside "extra" "args"}} />
+
+   {{!-- these will yield the modifier itself ("contextual modifier"), instead of invoking it now --}}
+
+   {{yield on-click-outside}}
```

Comment:
```suggestion
   {{!-- this will pass the modifier itself into the component, instead of invoking it now --}}
   <MyComponent @modifier={{modifier on-click-outside "extra" "args"}} />

   {{!-- this will yield the modifier itself ("contextual modifier"), instead of invoking it now --}}
   {{yield on-click-outside}}
```